Sources/Flash/Extensions/Request+Flash.swift

```diff
@@ -1,11 +1,25 @@
 import Vapor
 
-extension Request {
+public extension Request {
     @discardableResult
-    public func flash(_ type: Flash.Kind, _ message: String) -> Request {
-        if let container = try? privateContainer.make(FlashContainer.self) {
-            container.flashes.append(.init(type, message))
-        }
+    func flash(_ type: Flash.Kind, _ message: String) -> Request {
+        flashes.append(.init(type, message))
+
         return self
     }
+
+    var flashes: [Flash] {
```

Comment:
A protocol for this could be nice since both Request and Response have the same API.

Reply:
I'm not sure what advantage this might have? You can't declare conformances on extensions public, so neither Response or Request would benefit from any shared extensions I could write on the protocol itself.